boards/arm/stm32f103_mini/Kconfig.board

```diff
@@ -0,0 +1,8 @@
+# STM32F103RCT6 Mini board configuration
+
+# Copyright (c) 2020 WuhanStudio
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_STM32F103_MINI
+	bool "STM32F103RCT6 Mini Board"
+	depends on SOC_STM32F103XB
```

Comment:
According to HAL, that should actually be SOC_STM32F103XE (cf https://github.com/zephyrproject-rtos/hal_stm32/blob/master/stm32cube/stm32f1xx/soc/stm32f1xx.h#L73)

Reply:
Sorry, changed it.